src/argus_htmx/templates/htmx/incidents/_incident_ack.html

```diff
@@ -1,3 +1,5 @@
+{% load argus_htmx %}
+
 <a href="{% url 'htmx:incident-add-ack' pk=incident.pk %}">
-	{% if incident.acked %}Acked{% else %}Unacked{% endif %}
+    {% if incident|isackedby:field.context %}Acked{% else %}Unacked{% endif %}
```

Comment:
Yep, only way to do it in django. :thumbsup:

"is_acked_by" looks better though.